sdk/resources/mgmt/src/test/java/module-info.java

```diff
@@ -0,0 +1,8 @@
+module test.azure.resourcemanager.resources {
+    requires org.junit.jupiter.api;
+    requires azure.resourcemanager.resources;
+    requires com.azure.http.netty;
+    requires com.azure.identity;
+
+    exports com.azure.resourcemanager.resources.module.core;
```

Comment:
So `exports com.azure.resourcemanager.resources.core` does not work?

Reply:
Now it works. But it means we need second module-info under src/test/java. And to make it compile without errors, I exclude jakarta.activation from Azure.Core.

I will need to re-test after adding profile for java8 and java11.

Reply:
What I mean to say is, do you know why `exports com.azure.resourcemanager.resources.core` does not work but `exports com.azure.resourcemanager.resources.module.core` works?

Reply:
Got your question. The error message said packge `com.azure.resourcemanager.resources` exists in another module `azure.resourcemanager.resources`, which we export for ResourceManager already. So we need different package not exported by other modules.

Reply:
OK. I see. It complaining that same namespace is not allowed.

On 2nd thought, we do not exports `com.azure.resourcemanager.resources`, only `com.azure.resourcemanager.resources.core` which does not exists in main, which case seems similar as `com.azure.resourcemanager.resources.models.core` (which have common `com.azure.resourcemanager.resources.models`, but not exists in main as well)?